reledmac.dtx

```diff
@@ -1345,6 +1360,18 @@
 %   \item by \enquote{A-B} for the third lemma, as it starts while annotation A is still in effect and it finishes after annotation B has already taken effect;
 %   \item by \enquote{B} for the fourth lemma.
 % \end{itemize}
+%
+% Note that the annotation must be called before the beginning of a \cs{edtext} to be taken account as a starting annotation. So:
+% \begin{verbatim}
+% \linenumannotation{A}\edtext{something...\linenumannotation{B}}{\Afootnote{foo}}
+% \end{verbatim}
+% will use \verb+A+ as starting annotation, while:
+% \begin{verbatim}
+% \edtext{\linenumannotation{A}something...\linenumannotation{B}}{\Afootnote{foo}}
+% \end{verbatim}
+% won't.
+% \subsubsection{Resetting annotation}
+% You can manually reset annotation using \cs{resetlinenumannotation}\verb+{}+. Note that this macro does not take any argument, but you should use empty brackets or \textbackslash in order to avoid gobbling spaces.\footnote{As for all \TeX\ macro without argument.}
```

Comment:
"does not take any argument" → "takes no arguments"

"macro without argument" → "macros with no argument"